apps/web/src/containers/AnalyticsView/hooks/use-network-toggle.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTheme } from '@mui/material';
+import { TokenIcon } from '@notional-finance/icons';
+import { useState } from 'react';
+import { Body } from '@notional-finance/mui';
+import { FormattedMessage } from 'react-intl';
+
+export const useNetworkToggle = () => {
```

Comment:
Can we create a re-usable compoent for this? Isn't is used in multiple places?

Furthermore, instead of hardcoding the network options like this we should loop over SUPPORTED_NETWORKS and generate the options. That will be more scalable as we add more networks and then we dont have to come in and change all this stuff.